D1673/P1673.md

```diff
@@ -8938,6 +8935,8 @@ to all functions in this section.
 
   * [2.3]{.pnum} `C.extent(1)` equals `E.extent(1)` (if applicable).
 
+  * [2.4]{.pnum} The input matrix `A` is symmetric.
+
```

Comment:
This is not actually correct.  The input matrix does not need to be symmetric.  The Remarks below explain that the function only accesses one triangle (lower or upper) of the matrix, and interprets the other triangle so that the algorithm effectively works with a symmetric matrix.

```suggestion
```